.eslintrc.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
     ],
     'jest/expect-expect': 'off',
     'jest/no-disabled-tests': 'off',
+    'no-await-in-loop': 'off',
```

Comment:
tbh i don't understand this rule, it's nice to be able to await in a for loop, and we had it ignored in a variety of places so i've turned it off. lmk if this is controversial.